mcs/class/referencesource/mscorlib/system/security/cryptography/rc2cryptoserviceprovider.cs

```diff
@@ -63,7 +63,12 @@ public sealed class RC2CryptoServiceProvider : RC2 {
                 return m_use40bitSalt;
             }
             set {
+#if MONO
```

Comment:
That's the point: the usage of it is behind a !MONO ifdef so we're misleading the user here since it's never used. Another reason was that it made it easier to have same behavior for the CommonCrypto implementation :)